.github/workflows/lint.yml

```diff
@@ -7,20 +7,28 @@ on:
       - main
   pull_request:
 
+env:
+  CAIRO_VERSION: v1.0.0-alpha.7
```

Comment:
Let's use `alpha.6` for now since 7 has some breaking changes (it strips turbofish)